### PR TITLE
100% test coverage

### LINKED ISSUE #129: 💯 test coverage
I don’t know when and why exactly, but we dropped our 100% test coverage.

I’ve reduced coverall’s required coverage to 97% for the time being. Once fixed, we have to get it back to 100%. Setting is at https://coveralls.io/github/hoodiehq/hoodie-client/settings

See https://github.com/hoodiehq/hoodie-client/issues/79 for instructions how to run coverage locally

### PR DESCRIPTION
**Pull request by sohini-roy**
fixes #129

### CODE REVIEW

**Comment by gr2m** [comment]
Do you need any help? When you run `npm test` locally, you should see the same error as you currently get it on Travis

**Comment by sohini-roy** [comment]
Its a link error.I will give it another try

File: tests/specs/init.js [hunk 1]
```diff
@@ -53,17 +53,15 @@ test('"reset" triggered on "signin"', function (t) {
...
   beforeSignInCall.args[1](options)
 
   .then(function () {
     options.beforeSignin.accountId = 'accountid2'
     return afterSignInCall.args[1]({}, options)
   })
-
   .then(function () {
-    t.deepEqual(signInTestOrder, ['reset', 'connect'], 'store.connect was called after store.reset')
+    t.deepEqual(signInTestOrder, ['reset', 'connect'], 'store.con0nect was called after store.reset')
```
Comment (gr2m) [inline tests/specs/init.js]: `store.con0nect` should be `store.connect`

Comment (sohini-roy) [inline tests/specs/init.js]: Requested changes updated.

**Comment by gr2m** [comment]
Great work, thanks :)

**Pull request merged**